.github/ISSUE_TEMPLATE.md

```diff
@@ -1,23 +1,22 @@
-[Fill in the placeholders below. Delete any headings and placeholders that you do not fill in. Delete this line as well]
+**Operating System:** 
+**Commit/Build:** [e.g. 7b421c8 It can be found at the left bottom corner of the main menu.]
 
-**OS:** [e.g. Windows 10]
-**Version:** [e.g. 0.0.5]
-**Commit/Build:** [e.g. 426e106]
+**Explanation of the issue:**
 
-[Explanation of the issue...]
-
-- [ ] Reproducible in RCT2 (vanilla)?
-- [ ] Multiplayer?
+**Reproducible in RCT2 (vanilla)?:**
+**Multiplayer affected?:**
```

Comment:
@pizza2004 most people just used like `- [yes]`. It's hard to know that's checkbox and it's not meant to used that, too(it's for todo list)